src/containers/Comments/Comment.js

```diff
@@ -139,9 +139,9 @@ export const Comment = (props) => {
 
   const fetchComment = () => {
     return API.getComment(
-      comment.content_type,
-      comment.object_pk,
-      comment.id,
+      comment?.content_type,
```

Comment:
dumb question but can they be undefined / null or why do we need ? here ?

Reply:
actually good question and also Philli gave me a hint!
It is not needed as we assume if this component is going to be rendered, `comment` can't be undefined/null.

It was a leftover from developing.